Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -38,7 +38,11 @@ public enum SolverSafeZone
             /// <summary>
             /// Below where the controller meets the arm.
             /// </summary>
-            BelowWrist = 3
+            BelowWrist = 3,
+            /// <summary>
+            /// Floating above the palm.
+            /// </summary>
+            AtopPalm = 4
```

Comment:
If the hand pose changes to the palm facing you instead of up, then the safe zone could potentially be interpreted as floating in front of the palm instead of above.    Could add an extra note that this setting places the target object relative to palmJointPose.Up with an offset of safeZoneBuffer, might make things clearer.

Reply:
Good point, will fix.